test/data/mock-state.json

```diff
@@ -30,6 +30,25 @@
     "customTokenAmount": "10"
   },
   "confirm": {},
+  "confirmAlerts": {
```

Comment:
Is this required for the existing unit tests to pass or are we just adding alerts to the existing tests?

Is it safer to not update any existing tests for now since we're still developing and we can potentially create targeted tests instead?

It would also reduce the diff for this PR alot I think.

Reply:
I removed it and you are right in addition to the decoupling of the `InlineAlert` the diff ended up half of the previous size.